api/envoy/api/v2/core/health_check.proto

```diff
@@ -176,6 +176,9 @@ message HealthCheck {
   //
   // The default value for "healthy edge interval" is the same as the default interval.
   google.protobuf.Duration healthy_edge_interval = 16;
+
+  // Specifies the path to the health check event log.
```

Comment:
Can we cross link to relevant docs here about the event log, and also specify that if empty, no event log will be written?